brewtils/schemas.py

```diff
@@ -476,6 +480,27 @@ class JobSchema(BaseSchema):
     timeout = fields.Int(allow_none=True)
 
 
+class JobExportListSchema(BaseSchema):
+    jobs = fields.List(fields.Nested(JobSchema, allow_none=True))
```

Comment:
I mentioned this on the beer-garden side, but I think `JobExportSchema` is supposed to be the Nested field.

Reply:
Done, though it doesn't matter in practice, I agree the reader should understand the intent when looking at it.